pkg/generate/code/late_initialize.go

```diff
@@ -327,7 +345,11 @@ func IncompleteLateInitialization(
 				out += fmt.Sprintf("%sif ko.%s == nil {\n", indent, fNamePartAccesor)
 				fNameIndentLevel = fNameIndentLevel + 1
 				indent = strings.Repeat("\t", fNameIndentLevel)
-				out += fmt.Sprintf("%sreturn true\n", indent)
+				minBackoffSeconds := fieldNameToLateInitConfig[fName].MinBackoffSeconds
+				maxBackoffSeconds := fieldNameToLateInitConfig[fName].MaxBackoffSeconds
+				out += fmt.Sprintf("%sfDelay := (&acktypes.Exponential{Initial:time.Duration(%d)*time.Second, Factor: 2, MaxDelay: time.Duration(%d)*time.Second,}).GetBackoff(numLateInitializationAttempt)\n", indent, minBackoffSeconds, maxBackoffSeconds)
```

Comment:
Is there a reason why we are creating a pointer to the `acktypes.Exponential` type? Could this be handled with a method (`NewExponential`) perhaps?